bot/models/ignored_channel.py

```diff
@@ -0,0 +1,23 @@
+from .model import Model
+
+
+class IgnoredChannel(Model):
+    guild_id: int
+    channel_id: int
+
+    @classmethod
+    async def get(cls, guild_id: int):
+        query = """ SELECT * FROM ignored_channels WHERE guild_id = $1"""
+        return await cls.fetch(query, guild_id)
```

Comment:
`get` is usually used to get something from row, and often gets a single result
I'd rename this to `list_by_guild` or similar